src/components/WalletModal/index.tsx

```diff
@@ -2,8 +2,7 @@ import React, { useState, useEffect } from 'react'
 import ReactGA from 'react-ga'
 import styled from 'styled-components'
 import { isMobile } from 'react-device-detect'
-import { UnsupportedChainIdError } from '@web3-react/core'
-import { useWeb3React } from '../../hooks'
+import { UnsupportedChainIdError, useWeb3React } from '@web3-react/core'
```

Comment:
huh, what did this break?

Reply:
&& should we rename our version to not overlap?

Reply:
it was sometimes activating account connectors under the network key, which messed some stuff up. yeah, i'm down to rename, will do

Reply:
this explains the issues i was running into ln. thanks!